azure-pipelines.yml

```diff
@@ -31,6 +31,7 @@ pr:
       - main
       - dev
       - release/*
+      - validation/*
```

Comment:
Do we really want this to run for PRs against validation too? Or simply with manually queued builds?

Reply:
Good spot, I put this into the wrong trigger group, moved to the branches group.